connectors/concur/src/main/java/com/vmware/connectors/concur/ConcurController.java

```diff
@@ -86,6 +86,17 @@ public ConcurController(WebClient rest,
         this.concurrRequestTemplate = concurRequestTemplate;
     }
 
+    @GetMapping("/test-auth")
+    public Mono<ResponseEntity<Void>> verifyAuth(@RequestHeader(name = AUTHORIZATION_HEADER) final String authHeader,
+                                 @RequestHeader(name = BACKEND_BASE_URL_HEADER) final String baseUrl) {
+        return this.rest.head()
+                .uri(baseUrl)
+                .header(AUTHORIZATION, authHeader)
+                .retrieve()
+                .bodyToMono(Void.class)
```

Comment:
You can just return `Mono<Void>`; then you won't need to create the `ResponseEntity`.

Reply:
Changed the return type of test-auth endpoint to Mono<Void> in ```Concur``` and ```Bitbucket```.